### PR TITLE
✨ Add Adtelligent network to amp-ad

### PR DESCRIPTION
**Pull request by GeneGenie**
Adtelligent adNetwork

- added new type="adtelligent" to amp-ad

### CODE REVIEW

**Comment by GeneGenie** [comment]
@googlebot I signed it!

**Comment by GeneGenie** [comment]
@googlebot I signed it!

**Comment by GeneGenie** [comment]
@googlebot I signed it!

File: ads/adtelligent.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+-->
+
+# Adtelligent
+
+## Examples
+
+### For DSP/SSP Clients
+
+```
+<amp-embed width="300" height="250"
```
Comment (lannka) [inline ads/adtelligent.md:24]: do you want `amp-ad` or `amp-embed`?
usually all ad networks should go with `amp-ad`. content recommendation can use `amp-embed`. To use amp-embed, you will need to add an entry in AMP_EMBED_ALLOWED (integration.js)

File: ads/_config.js [hunk 2]
```diff
@@ -210,6 +210,10 @@ const adConfig = jsonConfiguration({
     preconnect: ['https://mads.at.atwola.com', 'https://aka-cdn.adtechus.com'],
   },
 
+  'adtelligent': {
+    preconnect: ['https://s.adtelligent.com'],
```
Comment (lannka) [inline ads/_config.js:214]: please try to implement renderStart API.

File: ads/adtelligent.md (hunk 1 above)
Comment (GeneGenie) [reply ads/adtelligent.md:24]: i'd like to stick with amp-ad

Comment (lannka) [reply ads/adtelligent.md:24]: please update this example then

File: ads/_config.js (hunk 2 above)
Comment (GeneGenie) [reply ads/_config.js:214]: @lannka Hello, sorry it took a long time to go back, but it's done

**Comment by lannka** [comment]
please fix the travis faliures.

**Comment by GeneGenie** [comment]
> please fix the travis faliures.

Fixed

**Comment by GeneGenie** [comment]
@lannka is this Percy issue fixable ?

**Comment by lannka** [comment]
@GeneGenie seems it stuck at the CLA bot. Did you sign as individual or corp?

**Comment by GeneGenie** [comment]
> @GeneGenie seems it stuck at the CLA bot. Did you sign as individual or corp?

I should be a part of corp. Adtelligent

**Comment by lannka** [comment]
@GeneGenie could you go to https://cla-assistant.io/ampproject to sign? The bot might be outdated as AMP switched to a new CLA rule.

**Comment by GeneGenie** [comment]
@googlebot I signed it!

**Comment by GeneGenie** [comment]
@lannka done

**Comment by lannka** [comment]
@GeneGenie do we still need this PR? if so pls help to address the merge conflict. (forgot where did we stuck, seems CLA check has been passed.)

**Comment by lannka** [comment]
likely a bad merge, touched 2k+ files. could you try again?

**Comment by GeneGenie** [comment]
yeah, sorry for that.
conflict resolved.
PR is still relevant

**Comment by GeneGenie** [comment]
@lannka It seems that test fail due to unrelated changes

**Comment by GeneGenie** [comment]
@lannka Hi, rebase everything helped

**Pull request merged**